includes/core/frontend/Get_Posts.php

```diff
@@ -652,6 +653,7 @@ public function autocomplete_search() {
 			}
 		}
 
+		$args = apply_filters('ymc_autocomplete_search_query_args', $args, $id, $term_ids);
```

Comment:
We will consider this in the future as an opportunity to change WP_Query settings.